hudi-spark/src/test/scala/TestDataSource.scala

```diff
@@ -135,6 +136,14 @@ class TestDataSource extends AssertionsForJUnit {
     countsPerCommit = hoodieIncViewDF2.groupBy("_hoodie_commit_time").count().collect();
     assertEquals(1, countsPerCommit.length)
     assertEquals(commitInstantTime2, countsPerCommit(0).get(0))
+
+    // pull the latest commit within certain partitions
+    val hoodieIncViewDF3 = spark.read.format("org.apache.hudi")
+      .option(DataSourceReadOptions.QUERY_TYPE_OPT_KEY, DataSourceReadOptions.QUERY_TYPE_INCREMENTAL_OPT_VAL)
+      .option(DataSourceReadOptions.BEGIN_INSTANTTIME_OPT_KEY, commitInstantTime1)
+      .option(DataSourceReadOptions.INCR_PATH_GLOB_OPT_KEY, "/2016/*/*/*")
```

Comment:
is the leading `/` necessary?  could we make it (if not already) such that the matchong works with or without it..

Reply:
Since we add `*` in front of it so it will still work without `/`. Here is just trying to be consistent with loading the full table.